go.mod

```diff
@@ -4,6 +4,11 @@ go 1.21
 
 require (
 	github.com/BurntSushi/toml v1.4.0
+	github.com/anacrolix/chansync v0.6.0
```

Comment:
How important is it to use these packages? We need to be really sure when adding new packages to be certain there aren't backdoors

Reply:
I definitely need chansync or I'd just end up duplicating all that again.

Some of the generic stuff is exceptionally useful for avoiding a lot of verbosity in Go.

The rest I was using for debugging.